docs/designs/cli.md

```diff
@@ -0,0 +1,80 @@
+# ElasticDL Command-line Client Tool
+
+## Background
+
+ElasticDL is a Kubernetes-Native deep learning framework. As it runs
+distributed training/prediction/evaluation jobs in a cluster, we need a client
+to submit the job to the cluster. Currently we have one but it's tightly
+coupled with the main package. User need pip install the whole elasticdl
+package and lots of dependencies and it's too heavy.
```

Comment:
It's too heavy that users need to install the elasticdl package with lots of dependencies.

Reply:
Good suggestion, updated.